showcase/src/assets/examples/slider/range-slider.html

```diff
@@ -0,0 +1,16 @@
+<div>
+	<h2>Range Slider</h2>
+	<stark-slider [values]="rangeSliderValues"
+				  sliderId="range-slider"
+				  [sliderConfig]="rangeSliderConfig"
+				  (changed)="onValuesChangeRangeSlider($event)">
+	</stark-slider>
+	<div>
+		<label>Value 1:</label>
+		<input type="number" [value]="rangeSliderValues[0]" (change)="onChangeRangeSlider($event, 0)">
+	</div>
+	<div style="margin-top: 8px;">
```

Comment:
We should avoid inline style. Better have a SCSS separate file and add it to the example-viewer